### ISSUE TITLE
bootstrap.php Error

### ISSUE THREAD

**Issue by ZapTechAdmin** [ZapTechAdmin/flarum-global-color-changer#1]
```
( ! ) Fatal error: Call to a member function listen() on null in C:\Program Files (x86)\Ampps\www\workbench\flarum-my-first-extension\bootstrap.php on line 5
Call Stack
#	Time	Memory	Function	Location
1	0.0004	130800	{main}( )	...\index.php:0
2	0.0167	698744	Flarum\Http\AbstractServer->listen( )	...\index.php:16
3	0.0327	1165488	Zend\Diactoros\Server->listen( )	...\AbstractServer.php:36
4	0.0327	1165520	Flarum\Http\AbstractServer->__invoke( )	...\Server.php:157
5	0.0327	1165952	Flarum\Foundation\AbstractServer->getApp( )	...\AbstractServer.php:49
6	0.1082	2849288	Flarum\Foundation\Application->register( )	...\AbstractServer.php:201
7	0.1095	2856576	Flarum\Extension\ExtensionServiceProvider->register( )	...\Application.php:375
8	0.2312	4364168	require( 'C:\Program Files (x86)\Ampps\www\workbench\flarum-my-first-extension\bootstrap.php' )	...\ExtensionServiceProvider.php:28
```

There is an error somewhere in the bootstrap.php that isn't allowing the extension to communicate with the custom .less code and the  .js files, causing a fatal error that breaks the forum.

**Comment by ZapTechAdmin** [comment]
Attempted to fix this with the latest commit, but to no avail.

**Comment by ZapTechAdmin** [comment]
Fixed with 51b5abb0f0080ec083b72fca6bd5fbae677f6efc
